src/main/views/courts/tabs/types.njk

```diff
@@ -0,0 +1,11 @@
+{% from "macros/csrf.njk" import csrfProtection %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+<div id='courtTypesTab'>
+  <input id='slug' type='hidden' value='{{court.slug}}'></input>
+  <form id="courtTypesForm" method="POST" action="/courts/{{slug}}/court-types">
+    {{ csrfProtection(csrfToken) }}
```

Comment:
Tom has implemented the CSRF protection now - it requires some additions to the controller for passing the token and checking the token.

Reply:
added correct CSRF implementations